Added SearchVersion to Workspace Properties

AErmie opened the pull request:
Added 'features' element within Properties to set Search Version to v2 (i.e. Kusto).

azurecla commented:
Hi __@elfisher__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

AErmie committed:
Added SearchVersion to Workspace Properties

Added 'features' element within Properties to set Search Version to v2 (i.e. Kusto).
